### PR TITLE
Fix/update readme

### PR DESCRIPTION
**Pull request by rgutzen**
- Fixing rst syntax
- improving the display of the cobrawap community by adding the logos of the involved institutions and grouping technical, scientific, and other supporting roles together (specific roles and contributions may be defined elsewhere in the acknowledgments) to highlight the breadth of the multidisciplinary and multinational community.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -74,34 +74,56 @@ Cobrawap is open-source software and is licensed under the `GNU General Public L
...
+- University of Milano (UniMi), Italy
+   - *Andrea Pigorini*
+   - *Thierry Nieus*
+   - *Gianluca Gaglioti*
+   - *Marcello Massimini*
 
 **Cobrawap Partnering Projects**:
 
-Sleep Wave Analysis Visualization Engine (SWAVE): A data visualization tool that takes Cobrawap outputs and visualizes these to show dynamic wave-like activity patterens found in the data. Developed at: https://github.com/cilantroxiao/SWAVE
+Sleep Wave Analysis Visualization Engine (SWAVE): A data visualization tool that takes Cobrawap outputs and visualizes these to show dynamic wave-like activity patterns found in the data. Developed at Washington University in St. Louis, USA:https://github.com/cilantroxiao/SWAVE
```
Comment (cosimolupo) [inline README.rst:126]:
```suggestion
Sleep Wave Analysis Visualization Engine (SWAVE): A data visualization tool that takes Cobrawap outputs and visualizes these to show dynamic wave-like activity patterns found in the data. Developed at Washington University in St. Louis, USA: https://github.com/cilantroxiao/SWAVE
```

File: README.rst [hunk 2]
```diff
@@ -74,34 +74,56 @@ Cobrawap is open-source software and is licensed under the `GNU General Public L
...
+|    fzj.svg                 |    infn.svg               |
+|    :height: 80px           |    :height: 80px          |
+|    :align: center          |    :align: center         |
+|    :width: 560px           |    :width: 560px          |
++----------------------------+---------------------------+
+| *Robin Gutzen* (now @ NYU),| *Giulia De Bonis*,        |
+| *Michael Denker*           | *Cosimo Lupo*,            |
+|                            | *Federico Marmoreo*,      |
+|                            | *Pier Stanislao Paolucci* |
++----------------------------+---------------------------+
```
[the remark this answers is not in the document]
Comment (rgutzen) [reply README.rst:94]: Thanks for noting. I added the full names back to the text with a new commit.

File: README.rst [hunk 3]
```diff
@@ -74,34 +74,56 @@ Cobrawap is open-source software and is licensed under the `GNU General Public L
...
```
Comment (mdenker) [inline README.rst:94]: I'd suggest to use bullets in the table, so names are better readable:
```suggestion
+------------------------------+-----------------------------+
| .. image::                   | .. image::                  |
|    doc/images/               |    doc/images/              |
|    institutions/             |    institutions/            |
|    fzj.svg                   |    infn.svg                 |
|    :height: 80px             |    :height: 80px            |
|    :align: center            |    :align: center           |
|    :width: 560px             |    :width: 560px            |
+------------------------------+-----------------------------+
| - *Robin Gutzen* (now @ NYU) | - *Giulia De Bonis*         |
| - *Michael Denker*           | - *Cosimo Lupo*             |
|                              | - *Federico Marmoreo*       |
|                              | - *Pier Stanislao Paolucci* |
+------------------------------+-----------------------------+
```

**Review by mdenker — APPROVED** [review]

File: README.rst [hunk 4]
```diff
@@ -74,34 +74,56 @@ Cobrawap is open-source software and is licensed under the `GNU General Public L
...
++------------------------------+-----------------------------+
+| - *Robin Gutzen* (now @ NYU) | - *Giulia De Bonis*         |
+| - *Michael Denker*           | - *Cosimo Lupo*             |
+|                              | - *Federico Marmoreo*       |
+|                              | - *Pier Stanislao Paolucci* |
++------------------------------+-----------------------------+
   
-Other people involved (past and present):
-Anna Letizia Allegra Mascaro (LENS), Irene Bernava (INFN), Cristiano Capone (INFN), Alessandra Cardinale (INFN), Miguel Dasilva (IDIBAPS), Chiara De Luca (INFN), Gianluca Gaglioti (UniMi), Arnau Manasanch (IDIBAPS), Marcello Massimini (UniMi), Maurizio Mattia (ISS), Thierry Nieus (UniMi), Francesco S. Pavone (LENS), Andrea Pigorini (UniMi), Francesco Resta (LENS), Maria V. Sanchez-Vives (IDIBAPS). 
+The further **Cobrawap Community** includes people and partners that offer technical support for the integration of the software in a larger framework of interoperable tools, and offer scientific support for the development of the analysis methods and the tool's integration into broader research endeavors.
```
Comment (cosimolupo) [inline README.rst:96]: Here is a remaining typo, *offer* should in fact be *offers*.

**Pull request merged**